### DISCUSSION TITLE
Embed in the program

### DISCUSSION THREAD

**Discussion by ghost** [yggdrasil-network/yggdrasil-go#915] [Q&A]
-  Accessing Yggdrasil network sites with curl (Solved)
- Are there any APIs I can build into a program in a language like Golang to make requests to Yggdrasil sites? That is, do not configure the yggdrasil binary, but embed it in the code, and also not to use sudo

**Comment by zhoreeq** [comment]
Use wget or curl lmao

**Reply by ghost** [reply]
> Use wget or curl lmao

Thanks. This does not work
<details>
 <summary>Screenshot</summary>

![i](https://user-images.githubusercontent.com/102379033/167680419-e7e1511b-f8cd-42c8-a16c-5fd55039fa2f.png)

</details>

**Reply by doosom** [reply]
Are you connected to any peers to reach that IP?

yggdrasilctl getpeers

if not, try adding some public peers from here

https://github.com/yggdrasil-network/public-peers

**Reply by ghost** [reply]
@doosom 
Yes, you are right. The issue with accessing the site from the console was solved by changing the config
